corehq/apps/cloudcare/static/cloudcare/js/formplayer/menus/views.js

```diff
@@ -616,6 +616,12 @@ hqDefine("cloudcare/js/formplayer/menus/views", [
                 FormplayerFrontend.trigger("menu:show:detail", this.options.model.get('id'), 0, false, true);
             }
         },
+        onAttach: function () {
+            $('.case-tile-container').css('margin-bottom', '12px');
```

Comment:
(not blocking) this seems hard to track down, since in the DOM inspector it'll just show up as an attribute of the element - does the margin have to go on the `.case-tile-container` or would it also work on the `#persistent-case-tile`?  If so, this could be defined by another class in `className` above, I think.